requirements.txt

```diff
@@ -1,4 +1,4 @@
-urllib3<1.24
-six==1.11.0
+urllib3<=1.24
```

Comment:
Also of note urllib3 <1.24 has a security issue; there is [another PR](https://github.com/transifex/transifex-client/pull/260) for this.
https://nvd.nist.gov/vuln/detail/CVE-2019-11324